packages/web/graphql/getPlayers.ts

```diff
@@ -194,3 +197,33 @@ export const getPlayerFilters = async (client: Client = defaultClient) => {
 
   return data;
 };
+
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+gql`
```

Comment:
```suggestion
/* GraphQL */ `
```

Reply:
Will be raising another PR  found a more better way to implement all this